### PR TITLE
Fixes wiki flagging system UI

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
closes https://github.com/EveripediaNetwork/issues/issues/676
closes https://github.com/EveripediaNetwork/issues/issues/732
closes https://github.com/EveripediaNetwork/issues/issues/708

# test
- Click the report to us text as an authenticated user.
- Click the report to us text as an unauthenticated user.

# Changes
- adds wiki flagging system UI
- fixes wierd syntax surrounding youtube vids
- hero card displays alt

### CODE REVIEW

**Review by Simplypheyie96 — COMMENTED** [review]
The pink stroke shouldn't be there. The text inside the box should be gray 800. and the pink color in dark mode is wrong
image

In dark mode, the same issues, the gray text shouldn't be on dark mode, the stroke colors are wrong
image

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
Nothing happens upon clicking confirm after making a report
wrong colors in dark mode
image
image
image

**Review by Simplypheyie96 — COMMENTED** [review]
Great work

**Comment by kesar** [comment]
mindblowing

**Review by Royal-lobster — APPROVED** [review]

**Review by Softdev1 — CHANGES_REQUESTED** [review]
after login , the toast still flags please login to continue which i showed you already

**Comment by Royal-lobster** [comment]
> 

@Softdev1 I pushed a fix for this, now it should allow flagging after login.

**Comment by Softdev1** [comment]
Awesome welldone @Adebesin-Cell  @Royal-lobster

**Comment by Adebesin-Cell** [comment]
Thanks @Simplypheyie96 and @Softdev1.

**Review by Softdev1 — APPROVED** [review]
This is great now 
welldone @Adebesin-Cell

File: src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+              fontWeight={500}
+              mt="3"
+              pb="3"
+              fontSize={{ base: '12px', md: '14px' }}
+              borderBottomWidth="1px"
+              borderBottomColor="gray.300"
+              color="gray.800"
+              _dark={{
+                borderBottomColor: 'whiteAlpha.300',
+                color: 'whiteAlpha.900',
```
Comment (anubra266) [inline src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx:92]: Let's use `semanticTokens` instead

File: src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+              onChange={e => {
+                setFlagContent(e.target.value)
+              }}
+              _dark={{ color: 'whiteAlpha.800', borderColor: 'whiteAlpha.200' }}
+            />
+          </ModalBody>
+          <ModalFooter justifyContent="center">
+            <Button
+              type="submit"
+              background="brand.500"
```
Comment (anubra266) [inline src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx:118]: Semantic tokens.
```suggestion
              background="brandLinkColor"

File: src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        mt={{
+          base: '5',
+          lg: '10',
+        }}
+      >
+        <Flex>
+          <Text as="span">See something wrong? </Text>
+          <Text
+            as="span"
+            ml="4px"
```
Comment (anubra266) [inline src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx:155]:
```suggestion
            ml="1"

File: src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+                  duration: 5000,
+                  isClosable: true,
+                })
+              }
+            }}
+            _dark={{ color: 'brand.800' }}
+          >
+            <Text as="span">Report to us.</Text>
+            <Flex as="span" ml="5px" alignItems="center">
+              <Icon fontSize={{ base: '16px', sm: '20px' }} as={RiFlagFill} />
```
Comment (anubra266) [inline src/components/Wiki/WikiPage/WikiFlaggingSystem.tsx:175]:
```suggestion
              <Icon fontSize={{ base: '4', sm: '5' }} as={RiFlagFill} />

**Comment by kesar** [comment]
👍👍👍 lets merge to show today

**Review by anubra266 — APPROVED** [review]

**Pull request merged**